packages/eslint-plugin-svelte/src/utils/index.ts

```diff
@@ -1,4 +1,50 @@
-import type { RuleModule, PartialRuleModule } from '../types.js';
+import type { RuleModule, PartialRuleModule, PartialRuleMetaData, RuleContext } from '../types.js';
+import { getSvelteContext, type SvelteContext } from '../utils/svelte-context.js';
+
+function isNotSatisfies<T>(actual: T, expected?: T[]): boolean {
+	if (expected == null || expected.length === 0) {
+		return false;
+	}
+
+	return !expected.includes(actual);
+}
+
+function satisfiesCondition(
+	condition: NonNullable<PartialRuleMetaData['conditions']>[number],
+	svelteContext: SvelteContext
+): boolean {
+	if (
+		isNotSatisfies(svelteContext.svelteVersion, condition.svelteVersions) ||
+		isNotSatisfies(svelteContext.fileType, condition.fileTypes) ||
+		isNotSatisfies(svelteContext.runes, condition.runes) ||
```

Comment:
`runes` are a boolean, `isNotSatisfies` doesn't work for that...

Reply:
I already changed it to array.